server/MusicCrawler.Lib/IRecommendationPersistanceRepo.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace MusicCrawler.Lib;
+
+public interface IRecommendationPersistanceRepo
+{
+    string GetEntireCollectionAsString(string collectionName);
```

Comment:
GetEntireCollectionAsString is unused and seems like a weird function to require all persistence alternatives to support.   What is a `collectionName`?  Feels like a MongoDB specific concept

Probably just remove for now since it's not used anyway?

Reply:
Yeah, I can clean that up.